### PR TITLE
Added tests for Tabs component

### PR DESCRIPTION
**Pull request by krolwofficial**

### CODE REVIEW

File: src/alto-ui/Tabs/__tests__/Tabs.test.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+import Tabs from '../Tabs';
+
+describe('Tabs', () => {
+
+    beforeEach(() => {
+        jest.clearAllMocks()
+    });
+
+    const getWrapper = (defaultProps, props) => mount(<Tabs {...defaultProps} {...props} />)
```
Comment (barteksiudak) [inline src/alto-ui/Tabs/__tests__/Tabs.test.js]: `defaultProps` should be always on top of the describe and this function should looks like:
```
const defaultProps = { ... };
const getWrapper = props => mount(<Tabs {...defaultProps} {...props} />);
```

and if you want to change something in `defaultProps` you should use it like:
```
describe('something', () => {
  let currentDefaultProps;
  beforeAll(() => {
    currentDefaultProps = { ...defaultProps, someProp: 'abc' };
  });
  it('is some test', () => {
    const wrapper = getWrapper({ someOtherProp: 'yoyo' });
  });
});
```

File: src/alto-ui/Tabs/__tests__/Tabs.test.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+    describe('renderLinks', () => {
+        const defaultProps = {
+            items: [{title: 'title_1', url: 'url_1'}],
+        }
+    
+        it('should render if value prop is not passed',() => {
+            const wrapper = getWrapper(defaultProps, {})
```
[the remark this answers is not in the document]
Comment (krolwofficial) [reply src/alto-ui/Tabs/__tests__/Tabs.test.js]: Refactored

File: src/alto-ui/Tabs/__tests__/Tabs.test.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+            expect(wrapper.find('.tabs__button').exists()).toBe(true);
+        })
+
+        it('should render with props',() => {
+            const props = {
+                id: 'id',
+                onChange: jest.fn(),
+            }
+
+            const wrapper = getWrapper(defaultProps, props)
```
[the remark this answers is not in the document]
Comment (krolwofficial) [reply src/alto-ui/Tabs/__tests__/Tabs.test.js]: Refactored: const tabsButton = getFirstButton(getWrapper());

File: src/alto-ui/Tabs/__tests__/Tabs.test.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+            expect(wrapper.find('.tabs__button').at(0).prop('id')).toBe('id__button--0')
+            expect(wrapper.find('.tabs__button').at(0).prop('className')).toBe('tabs__button tabs__button--active')
+            expect(wrapper.find('.tabs__button').at(0).prop('flat')).toBe(true)
+        })
+
+        it('should render button inner text from props',() => {
+            const wrapper = getWrapper(defaultProps, {})
```
[the remark this answers is not in the document]
Comment (krolwofficial) [reply src/alto-ui/Tabs/__tests__/Tabs.test.js]: Refactored

File: src/alto-ui/Tabs/__tests__/Tabs.test.js [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+                const props = {
+                    items: [{title: 'title_1', url: 'url_1', value: true}],
+                    onChange: mockFunc,
+                }
+
+                const wrapper = getWrapper(defaultProps, props)
+                const button = wrapper.find('.tabs__button').at(0)
+                button.simulate('click');
+
+                expect(mockFunc).toHaveBeenCalledTimes(0)
```
[the remark this answers is not in the document]
Comment (krolwofficial) [reply src/alto-ui/Tabs/__tests__/Tabs.test.js]: Refactored

**Pull request merged**